src/content/plugins/html-webpack-plugin.md

```diff
@@ -51,12 +52,9 @@ This will generate a file `dist/index.html` containing the following:
 </html>
 ```
 
-If you have multiple webpack entry points, they will all be included with `script`
-tags in the generated HTML.
+If you have multiple webpack entry points, they will all be included with `<script>` tags in the generated HTML.
 
-If you have any CSS assets in webpack's output (for example, CSS extracted
-with the [MiniCssExtractPlugin](/plugins/mini-css-extract-plugin/))
-then these will be included with `<link>` tags in the HTML head.
+If you have any CSS assets in webpack's output (for example, CSS extracted with the [MiniCssExtractPlugin](/plugins/mini-css-extract-plugin/)) then these will be included with `<link>` tags in the HTML head.
```

Comment:
If you have any CSS assets in webpack's output (for example, CSS extracted with the [MiniCssExtractPlugin](/plugins/mini-css-extract-plugin/)) then those will be included with `<link>` tags inside the `<head>` of HTML.

Reply:
i dont see a benefit of changing **in** to **inside**, but i like the idea of the rephrasing